src/main/scala-2.11/markets/clearing/engines/CDAMatchingEngine.scala

```diff
@@ -94,7 +94,7 @@ class CDAMatchingEngine(askOrdering: PriceOrdering[AskOrder],
           matches.enqueue(totalMatch)
         } else {  // incoming order is larger than existing order and will be rationed!
           val partialMatch = PartialMatch(askOrder, incoming, price)
-          val residualOrder = incoming.split(residualQuantity)
+          val (_, residualOrder) = incoming.split(residualQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1537948179/issues/source?bid=2921361&fileBranchId=3014278#l97)